internal/pkg/manifest/worker_svc.go

```diff
@@ -65,7 +65,7 @@ type SubscribeConfig struct {
 
 // IsEmpty returns empty if the struct has all zero members.
 func (s *SubscribeConfig) IsEmpty() bool {
-	return s.Topics == nil && s.Queue.IsEmpty()
```

Comment:
I feel like we still want to keep `s.Queue.IsEmpty()` -  the `IsEmpty()` should not return `true` if it is not really empty, i.e. has `queue`. Maybe instead, in validate, we can check the number of `s.Topics` and error out if there is none.